src/Examine.Lucene/Suggest/Directories/RAMSuggesterDirectoryFactory.cs

```diff
@@ -0,0 +1,9 @@
+using Lucene.Net.Store;
+
+namespace Examine.Lucene.Suggest.Directories
+{
+    public class RAMSuggesterDirectoryFactory : SuggesterDirectoryFactoryBase
```

Comment:
This is public and should therefore have XML documentation - Use `/// <inheritdoc />` where appropriate

Reply:
Done